apps/web/src/pages/integrations/IntegrationsStoreModal.tsx

```diff
@@ -136,6 +136,13 @@ export function IntegrationsStoreModal({
         <Grid.Col lg={12}>
           {!isLoading ? (
             <>
+              <ChannelGroup
+                selectedProvider={provider?.providerId}
+                channel={ChannelTypeEnum.IN_APP}
+                providers={inAppProvider}
+                title="In-App"
```

Comment:
```suggestion
                title="In-app"
```
I think when hyphenised the second word loses capitalisation. If it was `In App` with space would be ok. I also consider we need to define which is the term we will use throughout all the copies.